app/javascript/new_date.js

```diff
@@ -1,29 +1,29 @@
     window.onload = function() {changeDate()};
     
```

Comment:
whats with the indentation? Why not fix it?

Reply:
Did I make any change to this line?